config.toml

```diff
@@ -8,7 +8,7 @@
 version = 1
 
 [hoprd_node]
-endpoint = "http://123.456.7.89:3002"
+endpoint = "http://123.456.7.89:3001"
```

Comment:
Can you make these use the same sample IP as in the onboarding doc? which is: ` http://192.168.0.89:3001`